<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmFindAndReplace</class>
 <widget class="QDialog" name="FrmFindAndReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find And Replace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="gbFindAndReplace">
     <property name="title">
      <string>Find and Replace</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblFind">
        <property name="text">
         <string>Find:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnPrevious">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblReplace">
        <property name="text">
         <string>Replace:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtReplace"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnReplace">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnReplaceAll">
        <property name="text">
         <string>Replace All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
